public/assets/meetups/2024-09-19-lmt-03/data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "title": "Legnicki Meetup Technologiczny #3",
+  "datetime": "2024-09-19 00:15:00",
+  "place": "Legnicki Browar Książęcy",
+  "localization": "https://maps.app.goo.gl/MFfX39G6NrAerXWd6",
+  "lineup": [
+    {
+      "name": "Pxdddddd",
```

Comment:
Who ? :eyes: 

```suggestion
      "name": "Przemysław",
```

Reply:
It was by mistake; I added the file just to test things. I deleted this file